rest-api/test/unit_test/unit_test_util.py

```diff
@@ -577,7 +578,8 @@ def sort_lists(obj):
 
 
 def make_questionnaire_response_json(participant_id, questionnaire_id, code_answers=None,
-                                string_answers=None, date_answers=None, uri_answers=None):
+                                string_answers=None, date_answers=None, uri_answers=None,
+                                     consent_language=None):
```

Comment:
You might want to call this "language" rather than "consent_language"... it will be present on other questionnaire responses that aren't about consent.